### DISCUSSION TITLE
Does react with redux make sense for this?

### DISCUSSION THREAD

**Discussion by enn-code** [ionic-team/capacitor#3495] [Q&A]
Hi I'm new to cross-platform development and am a web developer. Normally I'd use React, Redux, Styled Components, etc. for a project but am not confident I should take exactly the same approach with this as state management for Ionic in general seems to be handled elsewhere.

Is using React with Redux recommended? Will it create unexpected performance issues?

Many thanks

**Comment by Sampath-Lokuge** [comment]
You can decide if you'll see this:  https://youtu.be/poQXNp9ItL4?t=531

**Reply by enn-code** [reply]
Sorry - I'm an experienced web developer and use redux regularly. Just wanted to know if it plays well with this platform or if there's some other way of doing state management that's more recommended/native :)

**Comment by imhoffd** [comment]
I think `@ionic/react` tries to make things more declarative so that components are less stateful, but I'm not sure it's 100% there. I'm also not an expert in Ionic Framework. This question might be better suited for the [Ionic forum](https://forum.ionicframework.com/). With Capacitor, of course you can use any state management library or any JS framework and even use something other than the Ionic Framework (or nothing at all!).
